### PR TITLE
add typed event for loading additional external storage backends

### PR DESCRIPTION
**Pull request by icewind1991**
Allow apps to load backends without using deprecated stuff

### CODE REVIEW

File: apps/files_external/lib/Events/LoadAdditionalStorageBackendsEvent.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ */
+
+namespace OCA\Files_External\Events;
+
+use OCP\EventDispatcher\Event;
+
+/**
+ * @since 22.0.0
+ */
+class LoadAdditionalStorageBackendsEvent extends Event {
```
Comment (rullzer) [inline apps/files_external/lib/Events/LoadAdditionalStorageBackendsEvent.php:31]: mmm same here. Maybe we should move it to OCP to have the clear separation of concerns... @ChristophWurst

**Comment by ChristophWurst** [comment]
We should document the old event as deprecated

**Comment by blizzz** [comment]
should move to 23?

**Comment by blizzz** [comment]
moving to 27

**Comment by skjnldsv** [comment]
As there is no feedback since a while I will close this ticket. 
If you will decide to work on this feature again and if it hasn't been fixed or implemented already, feel free to re-open and solve the various conflicts.

**Pull request closed without merging**